Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN apt-get update \
 
 WORKDIR /app
 
-CMD [ "/bin/bash", "-c", "bin/maxwell --user=$MYSQL_USERNAME --password=$MYSQL_PASSWORD --host=$MYSQL_HOST --producer=kafka --kafka.bootstrap.servers=$KAFKA_HOST:$KAFKA_PORT $MAXWELL_OPTIONS" ]
+CMD [ "/bin/bash", "-c", "bin/maxwell --user=$MYSQL_USERNAME --password=$MYSQL_PASSWORD --host=$MYSQL_HOST --producer=kafka --kafka.bootstrap.servers=$KAFKA_BROKERS $MAXWELL_OPTIONS" ]
```

Comment:
is there a way to add support for `KAFKA_BROKERS` without breaking existing boot scripts that rely on `KAFKA_HOST` and `KAFKA_PORT`?

Reply:
There is no change to do that.

Reply:
if you can't do it in docker, please add it to `bin/maxwell`

Reply:
https://kafka.apache.org/10/documentation.html
Here is the bootstrap.servers definition
A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. The client will make use of all servers irrespective of which servers are specified here for bootstrapping—this list only impacts the initial hosts used to discover the full set of servers. This list should be in the form host1:port1,host2:port2,....

Reply:
i see.